src/utils/calculations.ts

```diff
@@ -53,10 +53,12 @@ export const isAttestable: (currentApp: Application) => boolean = (currentApp) =
 };
 
 export const isRenewable: (currentApp: Application) => boolean = (currentApp) => {
-  // if the user has already started the renewal process, app state will be in DRAFT, SIGN AND SUBMIT, REVIEW or REVISIONS REQUESTED,
-  // so need to look for apps still in APPROVED or EXPIRED state, which means no action has been taken yet
-  // TODO: verify whether PAUSED apps can be renewed
-  if (!(currentApp.expiresAtUtc && ['APPROVED', 'EXPIRED'].includes(currentApp.state))) {
+  // can only renew an app in these states
+  if (!(currentApp.expiresAtUtc && ['APPROVED', 'EXPIRED', 'PAUSED'].includes(currentApp.state))) {
+    return false;
+  }
+  // can only create one renewal application per source application
+  if (currentApp.renewalAppId) {
```

Comment:
This is nitpicky but, your comment describes one half of this check, and the method is structured like it is doing multiple checks... maybe we should rewrite it as:

```js
// cannot renew an app in these states
if (!(['APPROVED', 'EXPIRED', 'PAUSED'].includes(currentApp.state)) {
    return false;
  }

// cannot renew an expired app
if(!currentApp.expiresAtUtc) {
  return false;
}
```

Reply:
looking at this again i'm not even sure i need that `expiresAtUtc` check 🤔

Reply:
Apps in those 3 states should always have an `expiresAtUtc` value, but i'll keep the check, i just moved it to before the date range verification